docs/src/stories/components/Layout/LayoutBeta.stories.jsx

```diff
@@ -0,0 +1,485 @@
+      table: {
+        category: 'Structure'
+      }
+    },
+    columnGap: {
+      options: ['none', 'normal', 'condensed'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Sets the gap between columns to distance them from each other. `normal` sets the gap to 16px, and to 24px on `lg` breakpoints and above. `condensed` keeps the gap always at 16px. `none` sets the gap to 0.',
+      table: {
+        category: 'Structure'
+      }
+    },
+    rowGap: {
+      options: ['none', 'normal', 'condensed'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Sets the gap below the header and above the footer. `normal` sets the gap to 16px, and to 24px on `lg` breakpoints and above. `condensed` keeps the gap always at 16px. `none` sets the gap to 0.',
+      table: {
+        category: 'Structure'
+      }
+    },
+
+    // Responsive
+
+    responsiveVariant: {
+      options: ['stackRegions', 'separateRegions'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: '`responsiveVariant` defines how the layout component adapts to smaller viewports. `stackRegions` presents the content in a vertical flow, with `pane` and `content` vertically arranged. `separateRegions` presents `pane` and `content` as different pages on smaller viewports. Change the preview size from the toolbar to test it.',
+      table: {
+        category: 'Responsive'
+      }
+    },
+    responsivePrimaryRegion: {
+      options: ['content', 'pane'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'When `responsiveVariant` is set to `separateRegions`, defines which region appears first on small viewports. `content` is default.',
+      table: {
+        category: 'Responsive'
+      }
+    },
+
+    // Pane
+
+    paneWidth: {
+      options: ['default', 'narrow', 'wide'],
+      control: {
+      type: 'inline-radio'
+      },
+      description: 'Defines the width of the pane',
+      table: {
+        category: 'Pane'
+      }
+    },
+    panePosition: {
+      options: ['start', 'end'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Defines the position of the pane. `start` renders the pane on the left, and `end` renders it on the right.',
+      table: {
+        category: 'Pane',
+      }
+    },
+    paneResponsivePosition: {
+      options: ['inherit', 'start', 'end'],
+      control: {
+        type: 'inline-radio',
+      },
+      description: 'If `responsiveVariant` is set to `stackRegions`, defines the position of the pane in the responsive variant. `start` puts the pane above `content`, and `end` puts it below `content`. `inherit` uses the same value from `panePosition`.',
+      table: {
+        category: 'Pane'
+      }
+    },
+    paneDivider: {
```

Comment:
Non-blocking nitpick: When I have a boolean prop, I usually like the names to sound like a boolean. For example `hasPaneDivider` instead of `paneDivider`.

This is just a personal preference when I'm working with components, so please disregard this if you or other reviewers don't like this.

Reply:
@mperrotti fixed it! this is only visible in storybook -- the modifier classes do have `has` in the name ;)